tests/integration/test_model.py

```diff
@@ -527,7 +529,7 @@ def test_export_model(c, training_df, tmpdir):
     )
 
     assert (
-        pickle.load(open(str(temporary_file), "rb")).__class__.__name__
+        pickle.load(open(str(temporary_file), "rb")).estimator.__class__.__name__
```

Comment:
Since `wrap_predict` will default to true for this case, we have a `GradientBoostingClassifier` wrapped in a `ParallelPostFit`, so we have to grab the estimator.